source/classification2.md

```diff
@@ -514,12 +623,13 @@ cancer_test_predictions = cancer_test.assign(
 cancer_test_predictions[['ID', 'Class', 'predicted']]
 ```
 
-### Compute the accuracy
+### Evaluate performance
 
 ```{index} scikit-learn; score
 ```
 
-Finally, we can assess our classifier's accuracy. We could compute the accuracy manually
+Finally, we can assess our classifier's performance. First, we will examine accuracy.
+We could compute the accuracy manually
 by using our earlier formula: the number of correct predictions divided by the total
 number of predictions. First we filter the rows to find the number of correct predictions,
 and then divide the number of rows with correct predictions by the total number of rows
```

Comment:
The row below says that we use the `len` function but we actually use the `shape` attribute

Reply:
We also do ` cancer_test.loc[:, ["Smoothness", "Concavity"]],` below instead of ` cancer_test[["Smoothness", "Concavity"]],`

Reply:
good catches, fixed